examples/01_standalone_sdk/03_activate_skill.py

```diff
@@ -46,19 +46,27 @@
 ]
 
 agent_context = AgentContext(
-    microagents=[
-        RepoMicroagent(
+    skills=[
+        Skill(
             name="repo.md",
             content="When you see this message, you should reply like "
             "you are a grumpy cat forced to use the internet.",
+            # source is optional - identifies where the skill came from
+            # (None = programmatically defined)
+            source="repo.md",
+            # trigger determines when the skill is active
+            # trigger=None means always active
+            trigger=None,
         ),
-        KnowledgeMicroagent(
+        Skill(
             name="flarglebargle",
             content=(
                 'IMPORTANT! The user has said the magic word "flarglebargle". '
                 "You must only respond with a message telling them how smart they are"
             ),
-            triggers=["flarglebargle"],
+            source="flarglebargle.md",
```

Comment:
```suggestion
            source=None,
```
Which also affects this one... 🤔 What do you think?